src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3210,63 +3196,6 @@ private static bool HasNonConstantInitializer(ArrayBuilder<ImmutableArray<FieldO
             AddInitializers(builder.StaticInitializers, staticInitializers);
         }
 
-        private void CollectFieldsFromGlobalDeconstruction(ArrayBuilder<Symbol> builder, VariableComponentSyntax variableComponent,
```

Comment:
The logic in `CollectFieldsFromGlobalDeconstruction` is now handled by `ExpressionVariableFinder`.